uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Assignment.php

```diff
@@ -21,4 +21,91 @@ public static function create($params) {
 
     return $instance;
   } */
+
+
+  /**
+   * Get all cases with no end dates
+   *
+   * @param array $params
+   * @param array $excludeCaseIds
+   * @param bool $excludeDeleted
+   *
+   * @return array of case and related data keyed on case id
+   */
+  public static function getUnclosedCases($params = array(), $excludeCaseIds = array(), $excludeDeleted = TRUE, $includeClosed = FALSE) {
```

Comment:
I think that, instead of just restoring the deleted code, we should take the this as an opportunity to improve things. Searching for the usage of the civicrm/case/ajax/unclosed endpoint, I could see it's only used by the AssignmentSearch service. This service is then used by the search method of the AssignmentService, which only accepts one filter (besides the search term) to of IDs to be excluded. With all that given, I believe this method should be simplified to reflect this and we should drop $excludeDeleted and $includeClosed. 
The $params parameter could also be simplified to only accept the sort_name param, which is the only one used by the AssignmentService.